### ISSUE TITLE
Title's can vanish

### ISSUE THREAD

**Issue by PhilCK** [vurtun/nuklear#264]
Observed on OSX 10.11 - SDL2 _(Edit also seen it in GLFW version)_

This is very bad inside my app, while not as bad - it is reproducible in the demo app.

_Steps for me in demo app are_
- Have all the demo windows.
- collapse 'overview' by touching the plus sign.
- make sure NodeEdit isn't overlapping.
- click on windows

<img width="459" alt="screen shot 2016-10-26 at 21 49 15" src="https://cloud.githubusercontent.com/assets/26706/19744718/94fefa86-9bc6-11e6-9814-031816d40632.png">

<img width="470" alt="screen shot 2016-10-26 at 21 49 23" src="https://cloud.githubusercontent.com/assets/26706/19744717/94fbb61e-9bc6-11e6-8943-8a6ebb082eda.png">

<img width="522" alt="screen shot 2016-10-26 at 21 50 53" src="https://cloud.githubusercontent.com/assets/26706/19744716/94fa83a2-9bc6-11e6-9cc8-112971cf3295.png">

**Comment by Jimmio92** [comment]
I've noticed this, also. I think it might be related to my issue. It seems the clipping is not working correctly. I fixed it temporarily by commenting out line 7452:
```c
nk_draw_list_add_clip(&ctx->draw_list, nk_rect(s->x, s->y, s->w, s->h));
```

but obviously this isn't optimal.

**Comment by PhilCK** [comment]
Yes that fixes the issue for me.

**Comment by Jimmio92** [comment]
As I said though, this is not a solution. There's a serious bug, obviously. Removing clipping can make for some pretty ugly stuff.

Totally off topic, I just saw the game Ride to Hell: Retribution. This bug now feels God-like and this piece of software is worth 20 pounds of dark matter. Heh.

**Comment by vurtun** [comment]
Hey, I found the bug and hopefully fixed it in commit: d8688c623f6c436a7f35617c85c62713c592f28c.
